src/views/common/Animation/AnimationControls/PlaybackOptionalButtons.ts

```diff
@@ -1,8 +1,9 @@
 
+import { BOOKMARK_BUTTON } from './PlaybackBookmarkButton'
 import { CROP_BUTTON } from './PlaybackCropWindowButton'
 import { SYNC_BUTTON } from './PlaybackSyncWindowButton'
 
-export type PlaybackOptionalButtons = typeof SYNC_BUTTON | typeof CROP_BUTTON
+export type PlaybackOptionalButtons = typeof SYNC_BUTTON | typeof CROP_BUTTON | typeof BOOKMARK_BUTTON
 
```

Comment:
Registering the new optional button with the infrastructure that manages such things.